drivers/SmartThings/matter-sensor/src/init.lua

```diff
@@ -86,15 +96,20 @@ local function illuminance_attr_handler(driver, device, ib, response)
 end
 
 local function temperature_attr_handler(driver, device, ib, response)
-  local temp = ib.data.value / 100.0
-  local unit = "C"
-  device:emit_event_for_endpoint(ib.endpoint_id, capabilities.temperatureMeasurement.temperature({value = temp, unit = unit}))
+  local measured_value = ib.data.value
+  if measured_value ~= nil then
```

Comment:
This isn't directly related to the pressure sensor and perhaps I should split it off into a separate PR, but the `MeasuredValue` attribute in the `TemperatureMeasurement` and `RelativeHumidityMeasurement` clusters are nullable so not having the check leads to a driver crash.